Lecture5/Task1/Task1L5.java

```diff
@@ -0,0 +1,42 @@
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+
+public class Task1L5 {
+
+    private static final String MESSAGE_NUMBER1 = "Enter the 1st number: ";
+    private static final String MESSAGE_NUMBER2 = "Enter the 2nd number: ";
+    private static final String MESSAGE_OPERATION = "Enter the operation (+ or -, * or /): ";
+    private static final String MESSAGE_DIVISION_BY_0 = "You cannot divide by 0";
+
+    private static BufferedReader userInputReader = new BufferedReader(new InputStreamReader(System.in));
+
+    public static void main(String[] args) {
+        while (true) {
+            double firstNumber = 0;
+            double secondNumber = 0;
+            char operation = ' ';
+            while (!Helper.InputProcessing.isInputAssigned()) {
+                System.out.print(MESSAGE_NUMBER1);
+                firstNumber = Helper.InputProcessing.getUserNumber(userInputReader);
```

Comment:
if this is main class who operates with inputAssigned boolean, this class should store and manipulate it. No need to have isInputAssigned variable

Reply:
FAD